webapp/src/components/ResourceMetadata.tsx

```diff
@@ -23,7 +44,31 @@ export default class ResourceMetadata extends React.Component<IPropTypes, never>
       title,
     } = this.props.resource;
 
+    const showConfirmArchiveResourceModal = () => this.setState({ modal: MODAL_TYPES.CONFIRM_ARCHIVE_RESOURCE });
+
+    const deleteSelectedResourceLocally = () => {
+      this.props.deleteResourcesLocally(Array(this.props.resource));
+      this.setState({modal: MODAL_TYPES.NONE});
+    };
+
+    const closeModal = () => this.setState({ modal: MODAL_TYPES.NONE });
+
+    
+
+    let modal;
+      switch (this.state.modal) {
+        case MODAL_TYPES.CONFIRM_ARCHIVE_RESOURCE:
+          const selectedArchResources = Array(this.props.resource);
+          modal = <ArchiveResourceConfirmationModal
+            close={closeModal}
+            resources={selectedArchResources}
+            submit={deleteSelectedResourceLocally}
+          />
+          break;
+      }
+    const archiveText = " This will delete a resource from your workspace but save it in HydroShare. Please manually transfer from your workspace any remaining files you'd like to save to HydroShare before archiving as all of your files in your workspace will be lost."
```

Comment:
again, see my earlier comment about the archive text

Reply:
I feel like, if we just called the button "Remove from Workspace", we could just put this message in the confirmation dialog (see above comments).

Reply:
yep sounds good